device/nokia/armhf-nokia_ixs7215_52x-r0/Nokia-7215/buffers.json.j2

```diff
@@ -2,7 +2,9 @@
 
 {% set default_cable = '40m' %}
 {% set default_ports_num = 54 -%}
+{}
```

Comment:
Why is this needed?

Reply:
the rendering code required some data to operate on when processing the jinja templates used by qos reload - the template I originally submitted had some small logic but did not return data to be parsed

Reply:
I see. Nitpicking here, but I think these braces should come after the comments below, since they should technically contain the lookup table defined in the comments.